<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxSettings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Timer interval</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxTimer">
        <property name="suffix">
         <string> min</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButtonEditShowWindowHotkey">
        <property name="text">
         <string>Show Window</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="HotkeyLineEdit" name="lineEditShowWindowHotkey"/>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pushButtonEditPauseTimerHotkey">
        <property name="text">
         <string>Pause Timer </string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="pushButtonEditResetTimerHotkey">
        <property name="text">
         <string>Reset Timer</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="HotkeyLineEdit" name="lineEditPauseTimerHotkey"/>
      </item>
      <item row="7" column="1">
       <widget class="HotkeyLineEdit" name="lineEditResetTimerHotkey"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Set Shortcut:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxShowWindowTimerEnd">
        <property name="text">
         <string>Show Window On Timer End</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBoxPinOnTop">
        <property name="text">
         <string>Pin On Top</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HotkeyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>HotkeyLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
